libcaf_net/test/endpoint_manager.cpp

```diff
@@ -106,9 +103,12 @@ class dummy_transport {
 
   template <class Manager>
   bool handle_write_event(Manager& mgr) {
+    auto sf = mgr.serialize_fun();
```

Comment:
Wait, why does `serialize_fun` still exist? If the endpoint manager does the serialization now, this should no longer exist. This callback was meant as a way to propagate serialization logic to the proxies (that did the serialization previously).